src/Compilers/Core/CodeAnalysisTest/Collections/List/SegmentedList.Generic.Tests.Misc.cs

```diff
@@ -1105,5 +1156,25 @@ public static void TrueForAllTests_Negative()
             intDriver.TrueForAll_VerifyExceptions(intArray);
             stringDriver.TrueForAll_VerifyExceptions(stringArray);
         }
+
+        [Fact]
+        public static void TrueForAll_ListSizeCanBeChanged()
+        {
+            List<int> list = new List<int>() { 1, 2, 3 };
+            List<int> expectedList = new List<int> { 1, 2, 3, 2, 3, 4, 3, 4, 4 };
```

Comment:
Should this be changed to SegmentedList?

Reply:
➡️ Yes, I amended the last commit (where a similar change was already being made to another test in the same file) to fix this.